opam/opam

```diff
@@ -5,17 +5,16 @@ maintainer: "seb@mondet.org"
 authors: [ "Sebastien Mondet <seb@mondet.org>" ]
 license: "Apache-2.0"
 build: [
-  ["omake"]
-  ["omake" "install" "BINDIR=%{bin}%"]
-]
-remove: [
```

Comment:
All solvuuu-build-based `opam` files use the `<project>.install` format to tell opam how to install/uninstall, so no need for a `remove` entry